doc/operation-and-maintenance/Logging-&-monitoring.md

```diff
@@ -38,7 +38,9 @@ For more advanced processing and analysis of logs, including gathering logs from
 
 ### WombatOAM
 
-WombatOAM is an operations and maintenance framework for Erlang based systems. Its Web Dashboard displays this data in an aggregated manner and provides interfaces to feed the data to other OAM tools such as Graphite, Nagios or Zabbix.
+WombatOAM is an operations and maintenance framework for Erlang based systems.
+Its Web Dashboard displays this data in an aggregated manner.
+What is more, WombatOAM provides interfaces to feed the data to other OAM tools such as Graphite, Nagios or Zabbix.
```

Comment:
Additionally, WombatOAM provides interfaces to feed the data to other OAM tools such as Graphite, Nagios or Zabbix.